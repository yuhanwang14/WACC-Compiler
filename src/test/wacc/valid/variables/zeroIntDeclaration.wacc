# simple zero integer variable declaration

# Output:

# Program:

begin
  int x = 0
end
