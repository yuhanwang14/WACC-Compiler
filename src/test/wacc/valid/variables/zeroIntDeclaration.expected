 1
 2
 3
 4
 5
 6
 7
 8
 9
10
# simple zero integer variable declaration

# Output:

# Program:

begin
  int x = 0
end
 